io.openliberty.java.internal_fat_misc/java17/LICENSE

```diff
@@ -0,0 +1,601 @@
+Code and build scripts are licensed under the Eclipse Public License v1
+
+Documentation files (e.g. files with an adoc file extension) are licensed under
+Creative Commons Attribution-NoDerivatives 4.0 International (CC BY-ND 4.0)
+
+Eclipse Public License - v 1.0
```

Comment:
The project root has a LICENSE file.  Not necessary to copy it into each sub-project.

Reply:
Good catch, it has been deleted.